applications/crossbar/src/crossbar_util.erl

```diff
@@ -867,12 +867,21 @@ create_auth_token(Context, Method, JObj) ->
     AccountId = wh_json:get_value(<<"account_id">>, JObj),
     OwnerId = wh_json:get_value(<<"owner_id">>, JObj),
 
+    Restrictions = case get_forced_restrictions(Method, AccountId, OwnerId) of
+        'undefined' ->
+            case wh_json:get_ne_value(<<"restrictions">>, Data) of
```

Comment:
No need to `case` the output here. `wh_json:get_ne_value/2` has a defaultt of `undefined`.

To be explicit you can use `wh_json:get_ne_value/3` with `undefined` as the last argument